plugins/solidigm/solidigm-nvme.h

```diff
@@ -0,0 +1,19 @@
+#undef CMD_INC_FILE
```

Comment:
Please add a `SPDX-License-Identifier: GPL-2.0-or-later` and copyright statement here. No need for the old style copyright header. (FWIW, I want to follow the kernel and have just the SPDX identifier at the beginning of each file)

Reply:
Done!